multicluster/controllers/multicluster/member_clusterset_controller.go

```diff
@@ -250,12 +240,20 @@ func (r *MemberClusterSetReconciler) updateStatus() {
 	status := multiclusterv1alpha1.ClusterSetStatus{}
 	status.TotalClusters = int32(len(r.clusterSetConfig.Spec.Members))
 	status.ObservedGeneration = r.clusterSetConfig.Generation
-	status.ClusterStatuses = r.remoteCommonAreaManager.GetMemberClusterStatues()
+	status.ClusterStatuses = []multiclusterv1alpha1.ClusterStatus{}
+	if r.remoteCommonArea != nil {
```

Comment:
We just need to lock from this line to line 251?

Reply:
You are right, changed the scope.